packages/artillery-plugin-publish-metrics/lib/open-telemetry/index.js

```diff
@@ -395,6 +424,117 @@ class OTelReporter {
     }
   }
 
+  tracePerformanceFlow(performanceEntries, specName, userFlowFunc) {
+    // Set all timing events we are interested in and map their start and end to their respective property names in the performanceEntry object
+    // We can then use this map to set spans for each of these events if their start and end times are present
+    const timingEventsMap = {
+      redirect: { start: 'redirectStart', end: 'redirectEnd' },
+      fetch: { start: 'fetchStart', end: 'responseEnd' },
+      DNS_lookup: { start: 'domainLookupStart', end: 'domainLookupEnd' },
+      TCP_handshake: { start: 'connectStart', end: 'connectEnd' },
+      TLS_negotiation: { start: 'secureConnectionStart', end: 'requestStart' },
+      Request: { start: 'requestStart', end: 'responseStart' },
+      Response: { start: 'responseStart', end: 'responseEnd' },
+      'DOM.content_loaded.event': {
+        start: 'domContentLoadedEventStart',
+        end: 'domContentLoadedEventEnd'
+      },
+      'load.event': { start: 'loadEventStart', end: 'loadEventEnd' }
+    };
+    // Set tracer for playwright
+    this.playwrightTracer = trace.getTracer('artillery-playwright');
+    debug('Tracer set');
+
+    // Set parent span startTime as first available timestamp
+    const firstAction = performanceEntries[0];
+    const parentStartTime =
+      firstAction.startTime ||
+      firstAction.redirectStart ||
+      firstAction.fetchStart ||
+      firstAction.requestStart;
+
+    // Start Parent Span
+    this.playwrightTracer.startActiveSpan(
+      specName || 'Scenario execution',
+      { kind: SpanKind.CLIENT, startTime: parentStartTime },
+      (parent) => {
+        performanceEntries.forEach((entry) => {
+          if (
+            entry.entryType != 'resource' &&
+            entry.entryType != 'navigation'
```

Comment:
I think it would be good to document the ones we are filtering and why. 

On that note, I believe that sometimes there may be a few additional [PerformanceEntry types](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry/entryType) we may be interested in here (when available), namely: `largest-contentful-paint`, `first-input` and maybe `layout-shift`.